moto/awslambda/models.py

```diff
@@ -131,6 +131,7 @@ def __enter__(self):
                 volumes = {self.name: {"bind": "/tmp/data", "mode": "rw"}}
             else:
                 volumes = {self.name: "/tmp/data"}
+            self._lambda_func.docker_client.images.pull("docker.io/library/alpine")
```

Comment:
Let's use the parse_image_name method here as well, i.e. `parse_image_name("alpine")` and `parse_image_name("lambci/lambda")`

Reply:
Done!